src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -157,15 +170,39 @@ private void HandleLocalResource(OpenDocumentationLinkEventArgs e)
         {
             try
             {
-                var targetContent = LoadContentFromResources(e.Link.ToString());
+                string targetContent;
+                Uri link;
+                switch (e)
+                {
+                    case OpenNodeAnnotationEventArgs openNodeAnnotationEventArgs:
+                        var mdLink = PackageDocManager.Instance.GetAnnotationDoc(openNodeAnnotationEventArgs.NodeNamespace);
+                        if (!(mdLink is null))
+                            WatchMdFile(mdLink);
```

Comment:
could this move to the `Link` setter just like Unsubscribe so they're in the same place ? 
had to jump around a bit to figure out the order they get called in

Reply:
Cant subscribe to the file on unsubscribe it in the same place? see this comment https://github.com/SHKnudsen/Dynamo/pull/9#discussion_r492219784

I might be over complicating this though! so happy to go through it at some point to explain why i did it like this.